main.py

```diff
@@ -81,7 +81,8 @@ def play_file(file_path):
             try:
                 frame = pipeline.run()
                 if wav_source.signal_info.sample_format in [16, 32]:
-                    # Convert back from float32 to int16/32
+                    # Convert back from float32 to int16/32
+                    # TODO: I think this belongs in the Pipeline class
```

Comment:
I guess so. We should have a discussion on the next steps and see how to organise the pipeline, because we could also have parallel audio circuits in the future for example :)

Reply:
Sure, that was my thoughts as well